content/docs/dvclive/how-it-works.md

```diff
@@ -73,43 +73,54 @@ model.pt.dvc
+
+Unlike other experiment trackers, DVCLive relies on Git to track the [directory]
+it generates, so it will save each run to the same path and overwrite the
+results each time. DVCLive uses Git to manage results, code changes, and data
+changes ([with DVC](#track-large-artifacts-with-dvc)). Include
 [`save_dvc_exp=True`](/doc/dvclive/live#parameters) to auto-track as a <abbr>DVC
-experiment</abbr>. DVC experiments are Git commits that DVC can find but that
-don't clutter your Git history or create extra branches.
+experiment</abbr> so you don't need to worry about manually making Git commits
+or branches for each experiment. You can recover them using `dvc exp` commands
+or using Git.
 
 ### Track large artifacts with DVC
 
 Models and data are often large and aren't easily tracked in Git.
-`Live.log_artifact("model.pt", type="model")` will
-[cache](/doc/start/data-management/data-versioning) the `model.pt` file with DVC
-and make Git ignore it. It will generate a `model.pt.dvc` metadata file, which
-can be tracked in Git and becomes part of the experiment. With this metadata
-file, you can [retrieve](/doc/start/data-management/data-versioning#retrieving)
-the versioned artifact from the Git commit.
-
-If `Live` was initialized with `dvcyaml=True` (which is the default) and you
-include values for any of the optional metadata arguments, this will add an
-[artifact](/doc/user-guide/project-structure/dvcyaml-files#artifacts) to the
-corresponding `dvc.yaml`. Passing `type="model"` will mark it as a `model` for
-DVC and will also show it in
-[Studio Model Registry](/doc/studio/user-guide/model-registry/what-is-a-model-registry).
+`Live.log_artifact("model.pt")` will [cache] the `model.pt` file with DVC and
+make Git ignore it. It will generate a `model.pt.dvc` metadata file, which can
+be tracked in Git and becomes part of the experiment. With this metadata file,
+you can [retrieve](/doc/start/data-management/data-versioning#retrieving) the
+versioned artifact from the Git commit. You can also use
+`Live.log_artifact("model.pt", type="model")` to add it to the [Studio Model
+Registry].
 
 Using `Live.log_image()` to log multiple images may also grow too large to track
 with Git, in which case you can use
 [`Live(cache_images=True)`](/doc/dvclive/live#parameters) to cache them.
 
-### Run with DVC
+### Customize with DVC
+
+DVCLive by default [generates] its own `dvc.yaml` file to configure the
+experiment results, but you can create your own `dvc.yaml` file to customize
+your project. For example, to define a [pipeline](#run-with-dvc) or
+[customize plots](/doc/user-guide/experiment-management/visualizing-plots#defining-plots).
+Do not reuse the DVCLive `dvc.yaml` file since it gets overwritten during each
+experiment run. Instead, write customizations to a new `dvc.yaml` file at the
+base of your repository or elsewhere outside the DVCLive directory.
+
+## Run with DVC
 
 Experimenting in Python interactively (like in notebooks) is great for
```

Comment:
are there any other benefits?

Reply:
There are more benefits listed later in the paragraph.

Reply:
yep, that's fine - it's just a bit abstract to me (as an end user). I mean the "more structured way to run
reproducible experiments" part and `parallelized hyperparameter search` jumps right into the advanced case. Again, I'm paying a lot of attention to this here since I expect the readers of this won't be DVC, and even not necessarily advanced Git users. There should be a story using their language / terminology as much as possible. Sorry, Dave for all this iterations. no intent to block it. I'm fine to merge it any time since it's an improvement already.

Reply:
Changed the examples here from `parallelized hyperparameter search` to `multi-step pipeline or queueing
multiple experiments`.